src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Hybrid development
+description: Hybrid development allows you to develop and run a service locally while the other components of your app run in the cloud.
+sidebar_label: Hybrid development
+id: hybrid-development
+---
+
+import Image from '@theme/Image';
+
+Okteto offers two different approaches to service development: *sync* ot *hybrid* mode. The command affected by this is the [`okteto up`](/reference/cli/#up) command being configurable at manifest level.
```

Comment:
```suggestion
Okteto offers two different approaches to service development: *sync* ot *hybrid* mode. The command affected by this is the [`okteto up`](reference/cli.mdx#up) command being configurable at manifest level.
```

You need to link directly to the .mdx file, else this will break when we do versioning.

More details here: https://github.com/okteto/docs/pull/295

Reply:
great 041bf0170a82d66651c665dd74636951a932d0bc